From e746aefb0f8c58cb8714e8954cd0b4a723c30094 Mon Sep 17 00:00:00 2001
From: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Date: Thu, 01 Sep 2016 22:04:59 +0300
[PATCH] drm: rcar-du: add ADV7513 HDMI encoder

The R-Car DU driver keeps a list of the external encoders which it uses
to  get  the encoder type.  Renesas Wheat board uses Analog Devices ADV7513
HDMI encoder, unlike the other Renesas boards which all use ADV7511W --
add it to the encoder list.

Signed-off-by: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
---
This patch is against the 'drm/next/du' branch of Laurent Pinchart's 'media.git'
repo...

 drivers/gpu/drm/rcar-du/rcar_du_kms.c |    1 +
 1 file changed, 1 insertion(+)

Index: media/drivers/gpu/drm/rcar-du/rcar_du_kms.c
===================================================================
--- media.orig/drivers/gpu/drm/rcar-du/rcar_du_kms.c
+++ media/drivers/gpu/drm/rcar-du/rcar_du_kms.c
@@ -362,6 +362,7 @@ static int rcar_du_encoders_init_one(str
 	} encoders[] = {
 		{ "adi,adv7123", RCAR_DU_ENCODER_VGA },
 		{ "adi,adv7511w", RCAR_DU_ENCODER_HDMI },
+		{ "adi,adv7513", RCAR_DU_ENCODER_HDMI },
 		{ "rockchip,rcar-dw-hdmi", RCAR_DU_ENCODER_HDMI },
 		{ "thine,thc63lvdm83d", RCAR_DU_ENCODER_LVDS },
 	};
